cluster/gce/addons/loadbalancing/cloud-provider-binding.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cloud-provider
```

Comment:
@tallclair Is it necessary to create the service account?

Reply:
Hmm, I think something is already creating it, but I'm not sure what.